profiler/src/main/java/com/splunk/opentelemetry/logs/OtlpLogsExporter.java

```diff
@@ -79,7 +87,31 @@ public void onSuccess(@Nullable ExportLogsServiceResponse result) {
 
     @Override
     public void onFailure(Throwable th) {
-      logger.debug("Failed to export logs", th);
+      errorLogger.log("Splunk profiling agent failed to export logs", th);
```

Comment:
In my opinion logging this with warn or error is enough, no need to bother with logging once an hour. Firstly isn't this class going to be replaced with logs exporter from otel? If so then all changes here would be lost when the switch happens. Secondly exporting logs should not be special cased. If exporting traces and metrics logs failures on every export then, for consistency, exporting logs should behave the same way or the same changes should be made to all exporters. Thirdly if there are intermittent failures this will allow logging such failure only once an hour, which would be confusing.

Reply:
Yeah, this is a valid point. I tried to point this out this same thing in #566. 
@gsmirnov-splk what do you think?

Reply:
I do not have a strong opinion here. I only suggested logging once per hour/per run because there were prior verbosity concerns cited.

Reply:
Ok, well I'm in favor of simpler/less code, so I'll just remove all the time-based guarding and just log at warn. We can relax it when folks complain.